app/model/orm/study.py

```diff
@@ -51,12 +51,14 @@ class Study(OrmBase):
     communities:  Mapped[List['Community']]   = owner_relationship()
     compartments: Mapped[List['Compartment']] = owner_relationship()
 
-    measurementTechniques:  Mapped[List['MeasurementTechnique']]  = owner_relationship()
-    measurementContexts:    Mapped[List['MeasurementContext']]    = owner_relationship()
-    modelingRequests:       Mapped[List['ModelingRequest']]       = owner_relationship()
-    experimentCompartments: Mapped[List['ExperimentCompartment']] = owner_relationship()
-    bioreplicates:          Mapped[List['Bioreplicate']]          = owner_relationship()
-    perturbations:          Mapped[List['Perturbation']]          = owner_relationship()
+    measurementTechniques:  Mapped[List['MeasurementTechnique']] = owner_relationship()
+    measurementContexts:    Mapped[List['MeasurementContext']]   = owner_relationship()
+    modelingRequests:       Mapped[List['ModelingRequest']]      = owner_relationship()
+
+    bioreplicates: Mapped[List['Bioreplicate']] = relationship(
+        secondary='Experiments',
+        viewonly=True,
+    )
```

Comment:
We no longer have a `studyId` in bioreplicates (at least on the model level), but we can still make an indirect relationship, `Study -> List['Experiment'] -> List['Bioreplicate']`.